readme.md

```diff
@@ -100,7 +100,7 @@ The `cart_total` value type of cart information represents as the number to avoi
 
 See the sample extension under `dist/chrome` after running build script. You can check the basic functionality of `fillr-extension` library on sample extension. After `load upacked`, all the page which has the form will be filled automatically.
 
-When you make your own extension, you should configure the following things on `manifest.json`. `all_frames` should be `true` and `js` includes the files which imports `@fillr_letspop/desktop-autofill`. This will enable all the iframe which has form to be filled.
+When you make your own extension, you should configure the following things on `manifest.json`. `all_frames` should be `true` and `js` includes the files which imports `@fillr_letspop/desktop-autofill`. This will enable all the iframe which has form to be filled. Also, `match_about_blank` should be `true` to inject the script in iframe having no url.
```

Comment:
When you make your own extension, you should configure the following things in your `manifest.json`. 
1. `all_frames` should be `true`.
2. `js` includes the files which imports `@fillr_letspop/desktop-autofill`. This will enable all the iframe which has form to be filled. Also, `match_about_blank` should be `true` to inject the script in iframe having no url.

Reply:
Updated!